front-end/src/app/shared/interfaces/transaction.interface.ts

```diff
@@ -6,4 +6,5 @@ export interface Transaction {
   transaction_id: string | null;
   transaction_type_identifier: string | null;
   contribution_purpose_descrip: string | null;
+  parent_transaction: number | null;
```

Comment:
Can we change this to parent_transaction_id for clarity? I see from the type it is an id number but just looking at the property name in the code and I wonder if it is the Transaction object itself